src/Simplex/Chat/Types.hs

```diff
@@ -1667,10 +1667,23 @@ instance ToJSON ChatVersionRange where
   toJSON (ChatVersionRange vr) = strToJSON vr
   toEncoding (ChatVersionRange vr) = strToJEncoding vr
 
+data ServerOperatorTag
+  = SOTSimplex
+  | SOTXyz
+  deriving (Show)
+
+data UsageConditionsAcceptance
+  = UCAAccepted {date :: UTCTime}
+  | UCAReviewAvailable {deadline :: UTCTime}
+  | UCAReviewRequired
+  deriving (Show)
+
 data ServerOperator = ServerOperator
   { operatorId :: Maybe OperatorId, -- it is Maybe to use this type to create new operators from the UI
     name :: Text,
-    preset :: Bool,
+    tag :: ServerOperatorTag,
```

Comment:
```suggestion
    tag :: Maybe ServerOperatorTag,
```

Reply:
Also I thought this type would include servers of the operator, and will be the only thing communicated to/from the apps.

The story of different servers per user is interesting, but I guess we can just have different enabled for preset still.